Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM debian:latest AS builder
+
+ENV ZCASH_URL=https://github.com/zcash/zcash.git \
+    ZCASH_CONF=/home/zcash/.zcash/zcash.conf
+
+ARG VERSION=master
```

Comment:
Just logging this following discussion at Zcon1. This should be moved below RUN statements and before  `RUN git checkout $VERSION` to prevent rebuilding these intermediate layers.